crates/iota-light-client/localnet.yaml

```diff
@@ -0,0 +1,7 @@
+rpc_url: "http://localhost:9000"
+graphql_url: "http://localhost:9125"
+# Provide an existing directory that contains the genesis file of the localnet
+checkpoints_dir: "checkpoints_localnet"
```

Comment:
Can't comment in the correct file, but `light_client.yaml` is different here
It has `checkpoint_summary_dir: "checkpoints_dir"`

Reply:
@Alex6323 you removed the localnet.yaml, but my `light_client.yaml` is still there like I mentioned and different than testnet.yaml

Reply:
Saw this independently of your comment. Sorry, other issues more important than this one, but I think I just forgot to remove it properly.